README.md

```diff
@@ -1,97 +1,70 @@
-thus be largely reduced to MesaTEE framework alone.
-
-More details can be found in the following documents:
-* [Threat Model](docs/threat_model.md)
-* [Design](docs/design.md)
-* [Hybrid Memory Safety and Non-bypassable Security](docs/hms_and_nbsp.md)
-* [Case Studies](docs/case_study.md)
-
-## Getting Started
-
-* [How to Build](docs/how_to_build.md)
-* [How to Run](docs/how_to_run.md)
-* [FAQs in Build and Run](docs/faq.md)
-* [Application Examples](examples/README.md)
-* [How to Add A Function](docs/how_to_add_your_function.md)
-* [How to Test](tests)
-* [Repository Structure](docs/repo_structure.md)
-* [Dependencies Vendoring](third_party/README.md)
-* [Blogs about MesaTEE](https://medium.com/@baidu.xlab)
+# Teaclave: Universal Secure Computing Platform
+
+Apache Teaclave (incubating) is an open source *universal secure computing*
+platform.
+
+Teaclave adopts multiple security technologies to enable secure computing, in
+particular, Teaclave uses Intel SGX to serve the most security-sensitive tasks
+with *hardware-based isolation*, *memory encryption* and *attestation*.
+Also, Teaclave is built in the Rust programming language to prevent
+*memory-safety* issues.
+
+Teaclave is provided as a *function-as-a-service platform* for secure computing.
+With many useful built-in functions, it supports tasks such as machine learning,
+private set intersection (PSI), crypto computation, etc. Developers can easily
+deploy a Python script in the Teaclave's trusted execution environment. More
+importantly, unlike traditional FaaS, Teaclave supports both general secure
+computing tasks and *flexible multi-party secure computation*.
+
+Teaclave builds its components in containers, therefore, it supports deployment
+both locally and within cloud infrastructures. Teaclave also provides client
+SDKs and command line tool.
+
+Teaclave is originated from Baidu X-Lab (formerly named MesaTEE).
+
+## Quickstart
 
-## Contributing
 
-The open-source version of MesaTEE is a prototype. The code is constantly
-evolving and designed to demonstrate types of functionality.
+Clone and build Teaclave services, examples, SDK, and command line tool.
 
-We still have lots of working-in-progress tasks. We are very happy if you are
-interested to submit pull requests. Please refer to
-[Issues](https://github.com/mesalock-linux/mesatee/issues) to help out
-or report new bugs/suggestions.
+```
+git clone https://github.com/apache/incubator-teaclave.git
+docker run --rm -v$(pwd)/incubator-teaclave:/teaclave -w /teaclave -it teaclave/teaclave-build-ubuntu-1804:latest
+mkdir -p build && cd build
+cmake -DTEST_MODE=ON .. && make
+```
 
-Please adhere to the [Rust Development Guideline](docs/rust_guideline.md) and remember to ``make format`` before submitting PRs.
+Start Teaclave services with `docker-compose` and detach into background. Make
+sure SGX driver and PSW package are properly installed and you have got the SPID
```

Comment:
Can we link to a reference doc about downloading/installing the driver, and apply for the IAS_KEY/IAS_SPID at https://api.portal.trustedservices.intel.com/EPID-attestation ?

Reply:
Updated.

I cannot find a good reference for installing driver and PSW. Will create one later in our own docs.